src/tir/schedule/primitive/cache_read_write.cc

```diff
@@ -1516,6 +1565,15 @@ StmtSRef CacheRead(ScheduleState self, const StmtSRef& block_sref, int read_buff
   }
 
   // Step 4. Making new cache stage block and rewrite readers.
+  info.cache_region = cache_region;
```

Comment:
just curious what if there's thread binding in outer scope
if we have threadIdx.x, it needs to be relaxed and the cache shape should be larger

Reply:
`AnalyzeRegionUpperBound` should have taken this into account.